go.mod

```diff
@@ -6,41 +6,46 @@ toolchain go1.22.7
 
 require (
 	github.com/Layr-Labs/eigenda v0.9.1-0.20250520184231-dbc01a215a28
-	github.com/Layr-Labs/eigenda-proxy/clients v0.0.0-00010101000000-000000000000
+	github.com/Layr-Labs/eigenda-proxy/clients v1.0.1
 	github.com/Layr-Labs/eigensdk-go v0.2.0-beta.1.0.20250118004418-2a25f31b3b28
 	github.com/avast/retry-go/v4 v4.6.0
-	github.com/consensys/gnark-crypto v0.12.1
+	github.com/consensys/gnark-crypto v0.16.0
 	github.com/ethereum-optimism/optimism v1.9.5
-	github.com/ethereum/go-ethereum v1.14.11
+	github.com/ethereum/go-ethereum v1.15.3
 	github.com/go-redis/redis/v8 v8.11.5
 	github.com/gorilla/mux v1.8.0
 	github.com/joho/godotenv v1.5.1
-	github.com/minio/minio-go/v7 v7.0.80
-	github.com/prometheus/client_golang v1.20.5
-	github.com/stretchr/testify v1.9.0
+	github.com/minio/minio-go/v7 v7.0.85
+	github.com/prometheus/client_golang v1.21.1
+	github.com/stretchr/testify v1.10.0
 	github.com/testcontainers/testcontainers-go/modules/minio v0.33.0
 	github.com/testcontainers/testcontainers-go/modules/redis v0.33.0
 	github.com/urfave/cli/v2 v2.27.5
 	github.com/wealdtech/go-merkletree/v2 v2.6.0
 	go.uber.org/mock v0.4.0
 	golang.org/x/exp v0.0.0-20241009180824-f66d83c29e7c
-	google.golang.org/grpc v1.64.1
+	google.golang.org/grpc v1.69.4
 )
 
 // TODO: Remove this after we have published v0.1.0 of the new clients module.
 replace github.com/Layr-Labs/eigenda-proxy/clients => ./clients/
 
+replace github.com/ethereum-optimism/optimism => github.com/Layr-Labs/optimism v0.0.0-20250522074124-9251a34831a7
```

Comment:
TODO: wait until https://github.com/Layr-Labs/optimism/pull/45 merges and then update this commit

Reply:
[ef23999](https://github.com/Layr-Labs/eigenda-proxy/pull/393/commits/ef23999ba2f48f2abfaf85ef321ab29d091d6064)